lib/vanagon/platform/defaults/solaris-11-i386.rb

```diff
@@ -1,9 +1,38 @@
 platform "solaris-11-i386" do |plat|
+
```

Comment:
Would like to request a comment on the source of this certificate data, when it expires (if it does) and how to update it when needed.